### ISSUE TITLE
lerna bootstrap does not work with yarn 2+

### ISSUE THREAD

**Issue by fvoska** [infinum/datx#1098]
### Used libraries

all

### Library version(s)

2.4.6

### Environments with the issue

linux; node v16.17.1, npm 8.15.0, yarn 3.2.3

### Current behavior

`CONTRIBUTING.md` is outdated. Following the setup instructions runs into errors:

```
❯ yarn global add lerna
Usage Error: The 'yarn global' commands have been removed in 2.x - consider using 'yarn dlx' or a third-party plugin instead

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...
```

I then tried running lerna using npx, and even that failed for some reason:
```
❯ npx lerna bootstrap           
lerna notice cli v5.6.2
lerna info bootstrap root only
Unknown Syntax Error: Unsupported option name ("--mutex").

$ yarn install [--json] [--immutable] [--immutable-cache] [--check-cache] [--inline-builds] [--mode #0]
lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'datx'
lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'datx';
```

### Expected behavior

Users should not install lerna globally because there is a possibility of mismatch between global and local version. For example, repo mentions lerna v5, while globall lerna installation would install 6.

We should update npm scripts so that user can do everything through those and use local version of lerna, ensuring same behaviour for everyone and removing the need to install lerna globally

### Reproduction steps

```bash
1. Clone project anew
2. Follow CONTIRUBING.md setup instructions
```

**Comment by fvoska** [comment]
I tried updating local lerna version to latest (v6), same issue. Not sure what the solution is.

**Comment by fvoska** [comment]
It might be an issue when node is installed using NVM

**Comment by fvoska** [comment]
Checked, same issue happens when using nativelly installed node.

The issue is that `lerna bootstrap` internally runs yarn install with some additional flags. One of those flags is `--mutex` and that flag does not exist in yarn 2+. You can see that this flag is [mentioned in yarn classic docs (v1)](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-concurrency-and-mutex) but there is [no mention of this argument in 2+ docs](https://yarnpkg.com/cli/install#options).

Conclusion: if we want to use lerna, we have to use classic version of yarn.

@DarkoKukovec we should decide between yarn 3 and lerna, we can't have both.

**Comment by DarkoKukovec** [comment]
@fvoska check the updated document - we don't really need the bootstrap command

**Comment by fvoska** [comment]
Fixed via #1101, removing any mention of `lerna bootstrap` which turns out to be unnecessary
